deploy/classic-api/README.md

```diff
@@ -3,10 +3,10 @@
 To install `classic-api` to the development namespace in the kubernetes cluster:
 
 ```bash
-helm install ./ --name=classic-api --set=image.tag=latest \
+helm install ./ --name=classic-api --set=image.tag=some_tag \
   --tiller-namespace=development --namespace=development  \
   --set=ingress.host=development.arxiv.org \
-  --set=elasticsearch.host=vpc-arxiv-es-ouzkxhljg5enjwygjjtxmtqymq.us-east-1.es.amazonaws.com \
+  --set=elasticsearch.host=foo.es.amazonaws.com \
```

Comment:
I wonder if there is a way to put these hosts and ports to non-kub services into kubernetes so that every pod gets this an an env var?